src/portfft/common/logging.hpp

```diff
@@ -151,8 +141,9 @@ struct global_data_struct {
    */
   template <typename TFirst, typename... Ts>
   __attribute__((always_inline)) inline void log_message_impl(TFirst message, Ts... other_messages) {
-    s << message << " ";
-    log_message_impl(other_messages...);
+    s << message;
+    ((s << ' ' << other_messages), ...);
+    s << '\n' << sycl::stream_manipulator::flush;
```

Comment:
I've removed this code this it wasn't really necessary for the PR, but also this would add a space in front of all messages, which disproportionately upsets me!